.ci_support/linux_c_compilergcccxx_compilergxxpython2.7.yaml

```diff
@@ -15,7 +15,7 @@ docker_image:
 expat:
 - '2.2'
 glib:
-- '2.56'
+- '2.58'
```

Comment:
The new `cairo` and `glib` may fix the failure. Fingers crossed!

Reply:
I think ti is because you forgot to bump the build number for gstreamer

Reply:
https://github.com/conda-forge/gstreamer-feedstock/pull/19